.github/ISSUE_TEMPLATE_tmpl/1_broken_site.yml

```
@@ -20,8 +20,6 @@ body:
           required: true
         - label: I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
```

The sort order can help to find existing issues; possibly also suggest adding  `-label:duplicate,spam` to the search.
```suggestion
        - label: I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones** (sort by `Best match` is recommended). DO NOT post duplicates
```

---

Perhaps we should also mention filters, as they are almost a necessity to be able to find relevant issues in our issue tracker.

I've added some shortcuts in https://github.com/yt-dlp/yt-dlp/issues/3766 as a trial. Perhaps we could provide a shortcut for, say in a broken site issue, `label:site-bug` issues?